src/snek5000/output/history_points.py

```diff
@@ -34,7 +34,10 @@ def __init__(self, output=None):
                 f"Should be (< {params.oper.max.hist = }, == {params.oper.dim = })"
             )
 
-        self.path_file = Path(output.path_run) / f"{sim.info_solver.short_name}.his"
+        # temporary?
+        output.path_session.mkdir(exist_ok=True)
+
+        self.path_file = output.path_session / f"{sim.info_solver.short_name}.his"
```

Comment:
Maybe this is a better solution, as it allows different history points coordinates on restart. Let's keep this instead of my suggestion of symlinking.